components/shared_code/src/shared/initialization/database.py

```diff
@@ -1,21 +1,24 @@
 """Database initialization."""
 
-import os
-
 import pymongo
 
-DEFAULT_DATABASE_URL = "mongodb://root:root@localhost:27017"
+from shared.utils.env import getenv
 
 
-def client(url: str | None = DEFAULT_DATABASE_URL) -> pymongo.MongoClient:  # pragma: no feature-test-cover
+def client() -> pymongo.MongoClient:  # pragma: no feature-test-cover
     """Return a pymongo client."""
-    database_url = os.environ.get("DATABASE_URL", url)
+    # DATABASE_URL is deprecated, use the new environment variables if DATABASE_URL is not set
```

Comment:
Does it make sense to use `DeprecationWarning` or at least warning logging?

Reply:
Good point.